pybigquery/sqlalchemy_bigquery.py

```diff
@@ -54,6 +54,7 @@ def format_label(self, *args, **kwargs):
     'BYTES': types.BINARY,
     'TIME': types.TIME,
     'RECORD': types.JSON,
+    'NUMERIC': types.Numeric,
```

Comment:
It seems to me that this should be types.DECIMAL to be 100% clear. (Numeric is the base class for both float and decimal)